addons/common/fnc_getUISize.sqf

```diff
@@ -24,8 +25,6 @@ Examples:
 Author:
     Written by Deadfast and made CBA compliant by Vigilante
 ---------------------------------------------------------------------------- */
-
-#include "script_component.hpp"
```

Comment:
This doesn't match the default style. So I'd say it's okey that, that newline is gone?

Reply:
Yeah, not sure if you used my regex, but I specifically made it account for cases where there might be space and strip it out

Reply:
I used yours. But slightly modified `[\s\S]` instead of `.`